app/Providers/AppServiceProvider.php

```diff
@@ -26,6 +26,5 @@ public function register()
     public function boot()
     {
         Schema::defaultStringLength(191);
-        URL::forceScheme('https');
```

Comment:
Is there something I missing about why this is being removed?

Reply:
Yes, see the pull request description. Essentially this breaks most local development envs without HTTPS and isn't even needed in prod when `APP_URL` is set correctly in `.env` (in this case it should be `APP_URL=https://utrs-beta.wmflabs.org`)

Reply:
So this is not true. It is currently set to that and I can clearly access it on HTTP.

Reply:
@dqwiki Interesting, it has worked on my production Laravel apps. Anyways I removed it from this PR so this can be otherwise reviewed.